telemetry/2.0/README.adoc

```diff
@@ -0,0 +1,34 @@
+---
+title: "MicroProfile Telemetry 2.0"
+date: 2024-06-19
+summary: 
+---
+MicroProfile Telemetry adopts OpenTelemetry from the CNCF and adds MicroProfile specification integration for improved MicroProfile application observability.
```

Comment:
MicroProfile Telemetry adopts https://opentelemetry.io/[OpenTelemetry] from the https://www.cncf.io/projects/opentelemetry/[CNCF] and adds MicroProfile specification integration for improved MicroProfile application observability.